src/PerfView/Dialogs/RunCommandDialog.xaml.cs

```diff
@@ -544,8 +544,16 @@ private void OKButtonClick(object sender, RoutedEventArgs e)
                 // m_args.Zip = ZipCheckBox.IsChecked;
                 m_args.Message = MarkTextBox.Text;
 
-                // This also throws an exception if there are illegal chars.  
-                var fullPath = System.IO.Path.GetFullPath(App.CommandLineArgs.DataFile);
+                string fullPath;
+                try
+                {
+                    fullPath = System.IO.Path.GetFullPath(App.CommandLineArgs.DataFile);
+                }
+                catch (Exception ex)
+                {
+                    m_mainWindow.StatusBar.LogError("Invalid datafile '" + App.CommandLineArgs.DataFile + "'. " + ex.Message);
```

Comment:
❓ Can we still print the full exception text to the log?

Reply:
It's that the path is invalid (blank, invalid characters). It's not a bug. Is it useful to write out the .NET stack for that?

Reply:
If that's the only case where it's hit, then I'm fine with it either way.